### PR TITLE
I4496 Adding search tools to navigation

### LINKED ISSUE #4496: Can't search for search plugins at /firefox/search/
Search plugins type addons don't appear to have been fully implemented. There is no selection in the dropdown to search for them via https://addons.mozilla.org/de/firefox/search/ though https://addons.mozilla.org/de/firefox/search/?q=OpenThesaurus&type=search does work.  Also https://addons.mozilla.org/de/firefox/search-tools/ still goes to a legacy page.

### PR DESCRIPTION
**Pull request by tsl143**
Fixes #4496 

searchtool

### CODE REVIEW

**Comment by tsl143** [comment]
Cool, I will create a component to solve the SEO thing :)

**Comment by tsl143** [comment]
@tofumatt Updated the PR.

**Comment by tsl143** [comment]
@tofumatt , please review this.

File: src/amo/components/SearchFilters/index.js [hunk 1]
```diff
@@ -119,8 +123,8 @@ export class SearchFiltersBase extends React.Component {
     const { i18n } = this.props;
 
     return [
-      { children: i18n.gettext('Relevance'), value: 'relevance' },
-      { children: i18n.gettext('Recently Updated'), value: 'updated' },
+      { children: i18n.gettext('Relevance'), value: SEARCH_SORT_RELEVANCE },
```
Comment (tofumatt) [inline src/amo/components/SearchFilters/index.js]: Nice! 👍

File: src/amo/components/SearchFilters/index.js [hunk 2]
```diff
@@ -100,6 +103,7 @@ export class SearchFiltersBase extends React.Component {
     return [
       { children: i18n.gettext('All'), value: NO_FILTER },
       { children: i18n.gettext('Extension'), value: ADDON_TYPE_EXTENSION },
+      { children: i18n.gettext('Search Tool'), value: ADDON_TYPE_OPENSEARCH },
```
Comment (tofumatt) [inline src/amo/components/SearchFilters/index.js]: While "Themes" are much more popular/common than Search Tools, I'm with you and will stick with alphabetical order until UX yells at us 😉

File: src/amo/components/SearchTools/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+import Search from 'amo/components/Search';
+import {
+  ADDON_TYPE_OPENSEARCH,
+  SEARCH_SORT_RELEVANCE,
+} from 'core/constants';
+import { convertFiltersToQueryParams } from 'core/searchUtils';
+
+type Props = {|
+  filters: Object,
```
Comment (tofumatt) [inline src/amo/components/SearchTools/index.js]: Arguably this should use `import type { SearchFilters} from 'amo/components/AutoSearchInput';`... which I know is just an object but would make it easier to replace with a better type later.

That said I think we still have to hunt for other instances of filter without a type and that's probably not even the right place for that type, so 🤷‍♂️

File: src/amo/components/SearchFilters/index.js (hunk 2 above)
Comment (tsl143) [reply src/amo/components/SearchFilters/index.js]: Yes :)

**Pull request closed without merging**